modes/obs-rpm-build/fs_rules.lua

```diff
@@ -651,6 +669,8 @@ emulate_mode_rules = {
 		{dir = "/var", rules = emulate_mode_rules_var},
 		{dir = "/opt", rules = emulate_mode_rules_opt},
 
+		{dir = tools_root, use_orig_path = true, readonly = true},
```

Comment:
I'm not sure if this should be belong to the same commit as the clang changes.

Reply:
It's WIP, don't look at it

Reply:
that's gone